### DISCUSSION TITLE
[功能建议] 可能需要添加新的排除网址

### DISCUSSION THREAD

**Discussion by Tinhone** [the1812/Bilibili-Evolved#4880] [功能建议 / Ideas]
`https://www.bilibili.com/correspond/1/[一串像是哈希的字符]`
![image](https://github.com/user-attachments/assets/fe9242f6-d291-49d5-a6de-5b655729870b)

目前看到会在视频页顶栏以 Iframe 嵌入。
![image](https://github.com/user-attachments/assets/8efa711a-1982-4f20-85de-1c176e792a01)

**Comment by the1812** [comment]
看来是满足某种条件出现的? 我这里的位置是一个注释
![Snipaste_2024-09-08_23-49-12](https://github.com/user-attachments/assets/4de39159-afdb-4965-af10-0936c6773bd4)
